src/api/entities/Proposal/types.ts

```diff
@@ -28,12 +28,6 @@ export interface Metadata {
   totalVotes: number;
 }
 
-export interface Proposal {
-  call: Call;
-  metadata: Metadata;
-  state: ProposalState;
-}
-
```

Comment:
Why is this deleted?

Reply:
I added it at the beginning of the Proposal entity but we never used it